src/sleep/sleep.c

```diff
@@ -645,7 +645,9 @@ static int run(int argc, char *argv[]) {
 
         }
 
-        return r;
+        if (r < 0)
+                return log_error_errno(r, "Failed to execute operation: %m");
```

Comment:
hmm, unfortunately logging is all over the place here. 

execute() logs at LOG_ERR about most errors, but not at all about some others. execute_s2h() doesn't log beyond LOG_DEBUG.

I think the right fix here is to teach both functions to log on their own for the remaining cases, and not add this line here.